src/components/Theme/ThemeToggle/index.tsx

```
@@ -0,0 +1,32 @@
+"use client";
+
+import { MoonIcon } from "@/icons/MoonIcon";
+import { SunIcon } from "@/icons/SunIcon";
+import React, { useState } from "react";
+import { useTheme } from "next-themes";
+
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)